cql3/statements/select_statement.cc

```diff
@@ -1427,6 +1561,17 @@ std::unique_ptr<prepared_statement> select_statement::prepare(database& db, cql_
     ::shared_ptr<cql3::statements::select_statement> stmt;
     auto prepared_attrs = _attrs->prepare(db, keyspace(), column_family());
     prepared_attrs->fill_prepare_context(ctx);
+
+    // Used to determine if an execution of this statement can be parallelized
+    // using `forward_service`.
+    auto can_be_forwarded = [&] {
+        return selection->is_aggregate()        // Aggregation only
+            && selection->is_count()            // Only count(*) selection is supported.
+            && !restrictions->need_filtering()  // No filtering
+            && group_by_cell_indices->empty()   // No GROUP BY
+            && ctx.bound_variables_size() == 0; // No bound variables
```

Comment:
Why?

Reply:
Because `forward_request` lacks names & values of bound variables. I'll add them in the next version.